force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -117,10 +183,94 @@ export default class AccountModelSettings extends LightningElement {
         this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
     }
 
+    handleAdministrativeAccountNamingChange(event) {
+        // hide/show custom Admin Account naming input field
+        this.showCustomAdministrativeAccountNaming =
+            event.detail.value === this.labelReference.accountNamingComboboxCustomOption;
+
+        // add updated setting to hierarchySettingsChanges object
+        let hierarchySettingsChange = {
+            settingsType: "string",
+            settingsName: "Admin_Account_Naming_Format__c",
+            settingsValue: event.detail.value,
+        };
+
+        this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
+    }
+
+    handleAdministrativeAccountCustomNamingChange(event) {
+        console.log("admin custom name");
+        //console.log("setting Admin custom format: " + JSON.stringify(event));
+        let adminAccountCustomNamingFormat = event.detail.value;
+
+        if (event.detail.value === '""') {
+            // set Hierarchy Settings field to a blank value (not "")
+            adminAccountCustomNamingFormat = "";
+        }
+
+        // add updated setting to hierarchySettingsChanges object
+        let hierarchySettingsChange = {
+            settingsType: "string",
+            settingsName: "Admin_Other_Name_Setting__c",
+            settingsValue: adminAccountCustomNamingFormat,
+        };
+
+        this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
+    }
+
+    handleHouseholdAccountNamingChange(event) {
+        // hide/show custom HH Account naming input field
+        this.showCustomHouseholdAccountNaming =
+            event.detail.value === this.labelReference.accountNamingComboboxCustomOption;
+
+        // add updated setting to hierarchySettingsChanges object
+        let hierarchySettingsChange = {
+            settingsType: "string",
+            settingsName: "Household_Account_Naming_Format__c",
+            settingsValue: event.detail.value,
+        };
+
+        this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
+    }
+
+    handleHouseholdAccountCustomNamingChange(event) {
```

Comment:
@MatthewBlanski is it acceptable to have this method fire every time something is typed in the custom name format field?

Reply:
added timeout as discussed